docs/topics/about-intellij-idea-plugin.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+// * docs/idea-plugin-guide/master.adoc
+
+
+[id='about-intellij-idea-plugin_{context}']
+= About the {ProductShortName} plugin for IntelliJ IDEA
+
+The {ProductName} ({ProductShortName}) plugin for IntelliJ IDEA helps you migrate and modernize applications.
+
+The {ProductShortName} extension analyzes your projects using customizable rulesets, marks issues in the source code, provides guidance to fix the issues, and offers automatic code replacement, if possible.
+
+ifdef::getting-started-guide[]
+For more information about using the {ProductShortName} extension, see the {ProductShortName} link:{ProductDocVscGuideURL}[_Visual Studio Code Extension Guide_].
```

Comment:
Re: {ProductDocVscGuideURL} attribute. I don't know how you can possible remember these attributes. Wouldn't it be easier if you used something shorter and more intuitive, like `{vsc-guide-url}`?

Reply:
I always look at the attributes file anyway, but let me think about simplying the items there (would be another PR). .